model/Course.js

```diff
@@ -21,17 +21,16 @@ const courseSchema = new Schema({
   number: String,
   department: String,
   tags: [{ type: String }],
-  area: String,
+  areas: { type: String },
   credits: { type: Number, required: true },
   wi: { type: Boolean, default: false },
   taken: { type: Boolean, default: false },
   preReq: { type: Array },
   isPlaceholder: { type: Boolean, default: false },
   isTransfer: { type: Boolean, default: false },
   ratings: Array,
-  distribution_ids: [
-    { type: Schema.Types.ObjectId, ref: "Distribution", required: true },
-  ],
+  distribution_ids: [{ type: Schema.Types.ObjectId, ref: "Distribution" }],
```

Comment:
we might want to set this to default: []
otherwise this field might be undefined and cause error when trying to read it